### DISCUSSION TITLE
Failure on repackaging

### DISCUSSION THREAD

**Discussion by Malclir** [REAndroid/APKEditor#101] [Q&A]
Hey there, 

recently trying to unpack and repack an apk by adding a tag to the `AndroidManifest.xml` but I keep getting "Resource not found for: '@style/null', frameworks size=2 [android-34, android-34]" failures. I'm assuming this is something I'm doing wrong on my end. Any thoughts on how I could fix this?

**Comment by REAndroid** [comment]
You need to explain a bit more, what tag you added on `AndroidManifest.xml` ?
